bounties/BA122402.md

```diff
@@ -0,0 +1,47 @@
+---
+code: BA122402
+title: Smart Node UI Wireframes
+proposal-author: Patches
+creation-date: 2024-05-28
+external-status: open
+summary: Design wireframes that could be used by a future UI developer and graphic designer to create a front-end for Smart Node.
+reward: 
+  unit: USD
+  max: 2000
+  min: 2000
+repeatable: false
+expiry: July 15, 2024
+application-url: https://dao.rocketpool.net/t/round-12-call-for-bounty-applications-deadline-is-may-7/2919/3?u=shfryn
+skillsets:
+  - node-operation
+tags: 
+  - frontend
+contacts:
+  - Patches
+  - ShfRyn
+---
+
+## Summary 
+Design wireframes that could be used by a future UI developer and graphic designer to create a front-end for Smart Node.
+
+Skillsets: Wireframing, UX, Product Design, Node Operation
+
+The ideal contributor is an already-experienced Rocket Pool Node Operator with the above skills.
+
+The wireframe should be a ‘blue sky’ design, and seek to replicate 100% of the functionality of the current CLI, in an organized fashion. Ideally the organization of the UI will be reminiscent of the organization of the CLI commands.
+
+Must be under an open source media license.
+
+## Further Notes
+* If you have never run smart node before, run a testnet node and thoroughly familiarize yourself with the current user experience
+* Don’t include auth in the wireframe, it will most likely be handled by external tools.
+* No web3 integration should be necessary, as the backend has access to the node wallet’s private keys. However, leave the door open to the notion of using one in the future, so a node can be controlled by a hardware wallet via the UI.
+* Try to separate “read only” from “read/write” in the UI mockup, similar to how some commands currently are read only (eg rocketpool node status) and others are read/write (eg rocketpool minipool stake.
```

Comment:
Probably insist on it in milestone section.

Reply:
Should we be doing Milestones section for projects that do not have more than one step to completion?